schema/BioOntologySchema.mcf

```diff
@@ -0,0 +1,5206 @@
+Node: dcid:psimiID
+typeOf: schema:Property
+name: "psimiID"
+description: "The identifier for Proteomics Standards Initiative Molecular Interaction."
+rangeIncludes: schema:Text
+domainIncludes:dcs:InteractionTypeEnum,dcs:InteractionDetectionMethodEnum,dcs:InteractionSourceEnum
+
+Node: dcid:goID
+typeOf: schema:Property
+name: "goID"
+rangeIncludes: schema:Text
+domainIncludes: dcs:InteractionDetectionMethodEnum, dcs:InteractionTypeEnum, dcs:InteractionSourceEnum
+description: "Identifiers of the Gene Ontology database AmiGO 2."
+url: "http://amigo.geneontology.org/amigo/landing"
+
+Node: dcid:residID
+typeOf: schema:Property
+name: "residID"
+rangeIncludes: schema:Text
+domainIncludes: dcs:InteractionDetectionMethodEnum, dcs:InteractionTypeEnum, dcs:InteractionSourceEnum
+description: "Identifiers of the RESID database which contains the collections of annotations and structures for protein modifications."
+url: "https://proteininformationresource.org/resid/"
+
+Node: dcid:InteractionTypeEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "InteractionTypeEnum"
+description: "The interaction type enumeration in biomedical ontologies."
+
+Node: dcid:InteractionDetectionMethodEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "InteractionDetectionMethodEnum"
+description: "The detection method enumeration in biomedical ontologies."
+
+Node: dcid:InteractionSourceEnum
+typeOf: schema:Class
+subClassOf: schema:Enumeration
+name: "InteractionSourceEnum"
+description: "The interaction source database enumeration in biomedical ontologies."
+
+Node: dcid:AffinityChromatographyTechnology
+typeOf: dcs:InteractionTypeEnum
+name: "AffinityChromatographyTechnology"
+psimiID: "MI:0004"
+description: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest (baits) and their binding partners. The baits can be captured by a variety of high affinity ligands linked to a resin - for example, antibodies specific for the bait itself, antibodies for specific tags engineered to be expressed as part of the bait or other high affinity binders such as glutathione resins for GST fusion proteins, metal resins for histidine-tagged proteins"
```

Comment:
For these descriptions, if you've written them yourself, this is good.

If these are copy/pastes from a source, then we add the URL string as value for the "descriptionUrl" property in this node.

Reply:
The descriptions are directly parsed from the database file itself. Should I add descriptionUrl: "https://www.ebi.ac.uk/ols/ontologies/mi" to all the nodes in this schema? The corresponding piece in the database file is as below:
[Term]
id: MI:0004
name: affinity chromatography technology
def: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest (baits) and their binding partners. The baits can be captured by a variety of high affinity ligands linked to a resin - for example, antibodies specific for the bait itself, antibodies for specific tags engineered to be expressed as part of the bait or other high affinity binders such as glutathione resins for GST fusion proteins, metal resins for histidine-tagged proteins." [PMID:7708014]
subset: PSI-MI_slim
synonym: "affinity chrom" EXACT PSI-MI-short []
synonym: "Affinity purification" EXACT PSI-MI-alternate []
is_a: MI:0091 ! chromatography technology
is_a: MI:0400 ! affinity technology

Reply:
description: "This class of approaches is characterised by the use of affinity resins as tools to purify molecule of interest..."
descriptionUrl: "https://www.ebi.ac.uk/ols/ontologies/mi"